src/PRBProxyRegistry.sol

```diff
@@ -247,35 +227,18 @@ contract PRBProxyRegistry is IPRBProxyRegistry {
 
     /// @dev See the documentation for the user-facing functions that call this internal function.
     function _deploy(address owner) internal returns (IPRBProxy proxy) {
-        // Load the next seed.
-        bytes32 seed = nextSeeds[tx.origin];
-
-        // Prevent front-running the salt by hashing the concatenation of "tx.origin" and the user-provided seed.
-        bytes32 salt = keccak256(abi.encode(tx.origin, seed));
+        // Use the address of the owner as the CREATE2 salt.
+        bytes32 salt = bytes32(abi.encodePacked(owner));
 
         // Deploy the proxy with CREATE2.
         constructorParams.owner = owner;
         proxy = new PRBProxy{ salt: salt }();
         delete constructorParams;
 
-        // Associate the the owner with the proxy in the mapping.
+        // Associate the owner and the proxy.
         _proxies[owner] = proxy;
 
-        // Increment the seed.
-        // We're using unchecked arithmetic here because this cannot realistically overflow, ever.
-        unchecked {
-            nextSeeds[tx.origin] = bytes32(uint256(seed) + 1);
-        }
-
         // Log the proxy via en event.
-        // forgefmt: disable-next-line
-        emit DeployProxy({
-            origin: tx.origin,
-            operator: msg.sender,
-            owner: owner,
-            seed: seed,
-            salt: salt,
-            proxy: proxy
-        });
+        emit DeployProxy({ origin: tx.origin, operator: msg.sender, owner: owner, proxy: proxy });
```

Comment:
Is it still neede to emit origin: tx.origin here?

Reply:
Nope.

https://github.com/PaulRBerg/prb-proxy/pull/123#discussion_r1238342252